### PR TITLE
fix(Column): Make Columns statically 12 on the base

### PR DESCRIPTION
**Pull request by codecaaron**
### Overview

Fixes a few future problems with LayoutGrid

**Default 12**

This fixes future bugs where size had not been configured as a base style.

```tsx
// Before
<Column size={{ xs: 12, md: 6 }} /> // This will be size 1 @media < 480px

// After
<Column size={{ xs: 12, md: 6 }} /> // 12 until @media 1024px
```

**Gutter Guards**
Previously LayoutGrid could potentially specify a column gap that would be able to exceed the size of their containers.

This adds a transform that sets the rule to: `min(calc(100% / 11), SIZE)`.  Unfortunately this will result in possible grid gaps that are not correctly scaled.  However on super small screens column gaps don't really apply as most items are 12 columns between 320 -> 480px.

* Preventing future confusion and bugs by making all columns have a default base size of 12 columns.  This will be overwritten by any props that sets size on the base media query.

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: ABC-123
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.41-alpha.f71b2f.0<br/>@codecademy/gamut-labs@14.2.1-alpha.f71b2f.0<br/>@codecademy/gamut@32.1.1-alpha.f71b2f.0<br/>@codecademy/styleguide@36.2.1-alpha.f71b2f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6080fe9da823d658aece7d18--gamut-preview.netlify.app>https://6080fe9da823d658aece7d18--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6080fe9da823d658aece7d18>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.41-alpha.8d9a6e.0<br/>@codecademy/gamut-labs@14.2.1-alpha.8d9a6e.0<br/>@codecademy/gamut@32.1.1-alpha.8d9a6e.0<br/>@codecademy/styleguide@36.2.1-alpha.8d9a6e.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60810720e86b026256a0dc87--gamut-preview.netlify.app>https://60810720e86b026256a0dc87--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60810720e86b026256a0dc87>Deploy Logs</a></p>

File: packages/gamut/src/Layout/LayoutGrid.tsx [hunk 1]
```diff
@@ -3,16 +3,31 @@ import { StyleProps, variance } from '@codecademy/variance';
 import styled from '@emotion/styled';
 import { pick } from 'lodash';
 
+const columns = 12;
+
 const gutters = pick(theme.spacing, [8, 16, 24, 32, 48]);
 
+/** This prevents grid blowut by ensuring that column gaps never exceed the size of their container */
```
Comment (JoshuaKGoldberg) [inline packages/gamut/src/Layout/LayoutGrid.tsx:10]:
```suggestion
/** This prevents grid blowout by ensuring that column gaps never exceed the size of their container */
```

**Review by JoshuaKGoldberg — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.41-alpha.bf2afb.0<br/>@codecademy/gamut-labs@14.2.1-alpha.bf2afb.0<br/>@codecademy/gamut@32.1.1-alpha.bf2afb.0<br/>@codecademy/styleguide@36.2.1-alpha.bf2afb.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60816f566ea64e3662f88e7f--gamut-preview.netlify.app>https://60816f566ea64e3662f88e7f--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60816f566ea64e3662f88e7f>Deploy Logs</a></p>

**Review by dreamwasp — APPROVED** [review]
✨

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6086ba15cf0f87cac67ad8c6--gamut-preview.netlify.app>https://6086ba15cf0f87cac67ad8c6--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6086ba15cf0f87cac67ad8c6>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.3.47-alpha.29636f.0<br/>@codecademy/gamut-labs@14.3.1-alpha.29636f.0<br/>@codecademy/gamut@32.2.1-alpha.29636f.0<br/>@codecademy/styleguide@36.3.1-alpha.29636f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6086bcb49bad65a2953ae784--gamut-preview.netlify.app>https://6086bcb49bad65a2953ae784--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6086bcb49bad65a2953ae784>Deploy Logs</a></p>

**Pull request merged**